ibm_concert/cert_renewal/create_USER_cert.yml

```diff
@@ -0,0 +1,82 @@
+###############################################################################
+# © Copyright IBM Corporation 2024
+###############################################################################
+
+- hosts: all
+  collections:
+    - ibm.ibm_zos_core
+  gather_facts: false
+  environment: "{{ environment_vars }}"
+  vars:
+    owner_id: 'STCUSR'
+    cert_label: 'concertCertificateTest'
+    sign_label: 'IBM CA'
+    cn: 'share.centers.ihost.com:19999'
```

Comment:
lets remove any specific system information for the sample going into `main`

Reply:
done